res/sapling-output-verifying-key.json

```diff
@@ -0,0 +1,16 @@
+{
+    "alphaG1": "0x0db882cf5db3e8567f16b4db1772d4d1f5a3fe8d62f0df2eb8a5cfa50806702afde8fc25335eb5ec859c2818b2610b2e19ab445dac720bb1f2b0cd3336f7a1acc62bf1b3a321826264dc7e469281e23b218394d598689da04e136878ff9a7897",
```

Comment:
`x` and `y` are merged here into single byte array - haven't found an API to construct points from separate `x` and `y`

Reply:
Yeah I had to alter `bn` back in 2017 library to do such API